config/zones/IE.yaml

```diff
@@ -48,13 +48,19 @@ emissionFactors:
       source: BEIS 2021
       value: 0.0
     coal:
-      datetime: '2021-01-01'
-      source: EU-ETS, ENTSO-E 2021
-      value: 758.4409086
+      - datetime: '2021-01-01'
+        source: EU-ETS, ENTSO-E 2021
+        value: 758.4409086
+      - datetime: '2022-01-01'
+        source: EU-ETS, ENTSO-E 2022
+        value: 1082.54
```

Comment:
We used data from 2 pps to compute this factor but the increase is important so we might want to investigate more